ember-cli-build.js

```diff
@@ -15,31 +15,6 @@ module.exports = function (defaults) {
       cascade: true,
       sourcemap: true,
     },
-    autoImport: {
```

Comment:
The addon already provides the autoImport config in the index.js file so this shouldn't be needed (unless something extra only for the dummy app is needed).